### PR TITLE
Support for escapeAttributeValues

### PR DESCRIPTION
**Pull request by codeclown**
Decided to pass the option to functions that need it, rather than store it in some global state for the time of the process.

Let me know if there's something to improve here. It's enjoyable to make a pull request to such a simple server-side library with great tests in place ;)

In regards to #23

### CODE REVIEW

**Comment by codeclown** [comment]
Typoed the last commit message, meant note, not node...

File: index.js [hunk 1]
```diff
@@ -85,7 +89,7 @@ function render(view) {
   }
 
   if (isArray(view)) {
-    return view.map(render).join('');
+    return view.map(function(view) { return render(view, options) }).join('');
```
Comment (StephanHoyer) [inline index.js]: there is one other place where render is called... se line 98

File: README.md [hunk 2]
```diff
@@ -15,6 +15,16 @@ var render = require('mithril-node-render');
...
 
+Options
+-------
+
+Optionally pass in options as an object: `m.render(component, options)`.
+
+The following options are supported:
+
+**escapeAttributeValues**  
+`Default: true`
```
Comment (StephanHoyer) [inline README.md]: maybe add a small description

File: index.js [hunk 3]
```diff
@@ -55,9 +55,10 @@ function createAttrString(attrs) {
           return styles[property] != '' ? [camelToDash(property).toLowerCase(), styles[property]].join(':') : '';
         }).filter(removeEmpties).join(';');
       }
-      return styles != '' ? ' style="' + escapeHtml(styles, true) + '"' : '';
+      return styles != '' ? ' style="' + (escapeAttributeValues ? escapeHtml(styles, true) : styles) + '"' : '';
```
Comment (StephanHoyer) [inline index.js]: I think the code would be much simpler if we allow to overwrite the escape-html function. So options with default values might be
```
{
  escapeAttributeValue: escapeHtml,
  escapeString: escapeHtml
}
```
This way the user has much better control over whats happening

**Comment by codeclown** [comment]
Great suggestions and notes, will update pull request!

**Comment by codeclown** [comment]
Sorry it took so long, but here's an update with filter functions.

**Pull request merged**